be/src/connector/hive_connector.cpp

```diff
@@ -674,7 +675,11 @@ Status HiveDataSource::get_next(RuntimeState* state, ChunkPtr* chunk) {
     return Status::OK();
 }
 
-void HiveDataSource::_init_chunk(ChunkPtr* chunk, size_t n) {
+void HiveDataSource::_init_chunk_if_needed(ChunkPtr* chunk, size_t n) {
```

Comment:
Should make Status as this function's return type.
Suppose if new_chunk will fail in the future, we need to return that error rather than void.

Reply:
done